add wireframe pictures and business case doc

sara-rioseco opened the pull request:
Thanks!

sara-rioseco committed:
add wireframe pictures and business case doc

spancavil commented:
Hi girls! Good job.
I haven't access to Figma (make it public if you can) The design pictures are ok. The documentation with the business case is also fine.
Remains to be done:
- Which dependencies will you use? Justify their use
- Mention if you will use typescript, prettier and ESLint.
- How are you going to give styles to your project?
I await this, thanks.

sara-rioseco committed:
Create LICENSE

sara-rioseco committed:
Update README.md

sara-rioseco commented:
Thank you @spancavil !
We've updated the README file and added all the missing information.
Figma file link should be available now for anyone to see, the wireframe doesn't have any interactions yet, though.

sara-rioseco committed:
Update README.md

Fix project url on readme

spancavil commented:
Awesome!
Just a little reminder: an admin can change all fields of a user, except RUT and password. A user can change his/her password only through the "change password" screen.
Good job girls! congratulations.
